js/src/layers/Path.js

```diff
@@ -27,19 +27,6 @@ export class LeafletPathModel extends vectorlayer.LeafletVectorLayerModel {
 export class LeafletPathView extends vectorlayer.LeafletVectorLayerView {
   model_events() {
     super.model_events();
-    var key;
-    var o = this.model.get('options');
-    for (var i = 0; i < o.length; i++) {
-      key = o[i];
-      this.listenTo(
-        this.model,
-        'change:' + key,
-        function() {
-          this.obj.setStyle(this.get_options());
```

Comment:
Did you check that this was still working? It's not exactly the same code.

Reply:
Good point, it doesn't work. It works if I keep `Path.js` as it were, but the `LeafletLayerView` still calls `L.setOptions` on the `LeafletPathView`. It doesn't seem to hurt, but is it okay?

Reply:
Let's keep it as it was then.. It is a bit annoying. The API of LeafletJS is too heterogeneous.

Reply:
Do you mean we should close this PR?

Reply:
No I mean let's keep `Path.js` as it was with the `setStyle`, but keep your other changes.

Reply:
:+1: